client/webserver/site/.eslintrc.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
   parser: '@typescript-eslint/parser',
   parserOptions: {
     project: './tsconfig.json', // Required to have rules that rely on Types.
-    tsconfigRootDir: './'
+    tsconfigRootDir: __dirname
```

Comment:
Can you comment on this change?

Reply:
This allows eslint to be run from the command line.